CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Features
+
+- New Grafana Agent Flow components:
+
+  - `discovery.aws.ec2` service discovery for aws ec2. (@captncraig)
+  - `discovery.aws.lightsail` service discovery for aws lightsail. (@captncraig)
```

Comment:
Do these need to have `aws` in the name instead of being called `discovery.ec2` and `discovery.lightsail`? Isn't AWS implied by ec2 and lightsail?

Reply:
Great idea.